episodes/17-live.md

```diff
@@ -34,11 +34,11 @@ and work through the lesson,
 typing in the code,
 reformatting data,
 and talking as we go.
-This is called ["live coding"](https://en.wikipedia.org/wiki/Live_coding).
+This is called **[live coding](https://en.wikipedia.org/wiki/Live_coding)**.
```

Comment:
flagging as a part of the quote vs italics discussion, here I think I am okay but we do need to check for consistency